extensions/amp-consent/0.1/consent-config.js

```diff
@@ -320,15 +322,21 @@ export class ConsentConfig {
  * Expand consent endpoint url
  * @param {!Element|!ShadowRoot} element
  * @param {string} url
+ * @param {Object<string, *>=} vars
  * @return {!Promise<string>}
  */
-export function expandConsentEndpointUrl(element, url) {
+export function expandConsentEndpointUrl(element, url, vars = {}) {
```

Comment:
This is minor, but: @dmanek suggested this change from `opt_vars` to `vars = {}` but I disagree.

Spreading `undefined` has no effect on the outer object, and we prevent creating an extraneous object from the default value. Please change back to `opt_vars` without a default value.